src/components/Comments/Comments.jsx

```diff
@@ -80,20 +80,18 @@ const Comments = () => {
     })
```

Comment:
Here are some links regarding useMemo - like I said it's completely optional, you can do the same thing with states and useeffects but will send for your info. it's essentially like a useeffect in terms of syntax, but you can return a value from it which can be saved to a variable, e.g. 

`const users = useMemo(()=>{
// write logic here and return a value here which will be accessible in 'users'
}, [anyDependenciesYouNeed])`


Links:
https://react.dev/reference/react/useMemo
https://www.w3schools.com/react/react_usememo.asp
https://www.joshwcomeau.com/react/usememo-and-usecallback/